src/service/packet_verifier.proto

```diff
@@ -18,3 +18,9 @@ message invalid_packet {
 enum invalid_packet_reason {
   invalid_packet_reason_insufficient_balance = 0;
 }
+
+message valid_data_transfer_session {
+  bytes pub_key = 1;
+  uint64 upload_bytes = 2;
+  uint64 download_bytes = 3;
```

Comment:
no session_id imsi_hash propagated? It seems impossible to trace back to the reported data transfer req that caused this output?

Reply:
If you can't charge for it because the balance fell too low would that not be an "invalid one"?